terraso_backend/apps/graphql/schema/users.py

```diff
@@ -54,8 +55,12 @@ class Input:
 
     @classmethod
     def mutate_and_get_payload(cls, root, info, **kwargs):
+        request_user = info.context.user
         _id = kwargs.pop("id")
 
+        if str(request_user.id) != _id:
+            raise GraphQLValidationException("User has no permission to change User.")
```

Comment:
issue: This messages is unclear. Perhaps indicate they cannot change a different user/someone else/a user other than themselves.